roles/iridl/tasks/docker-ce.yaml

```diff
@@ -0,0 +1,19 @@
+- name: install docker-ce repository
+  get_url:
+    url: https://download.docker.com/linux/centos/docker-ce.repo
+    dest: /etc/yum.repos.d/docker-ce.repo
+
+- name: install docker-ce
+  # latest stable versions as of 4/19/24
+  package:
+    name:
+      - docker-ce-3:26.0.2-1.el9
+      - docker-ce-cli-1:26.0.2-1.el9
+      - containerd.io-1.6.31-3.1.el9
+      - docker-buildx-plugin-0.14.0-1.el9
+      - docker-compose-plugin-2.26.1-1.el9
+  #    - python-requests
```

Comment:
If you've confimed that it's not needed then remove this line, don't leave it as a comment.